todo/src/Components/Todo.jsx

```diff
@@ -1,11 +1,11 @@
 import React, { Component } from 'react';
 
 
-function Todo(props) {
+function Todo({todo}) {
     return (
         <div>
-
-            {props.item}
+            {console.log(todo)}
```

Comment:
If this is logging for System Administration purposes, adding more detail (e.g. date/time stamp) is advisable.  If not, this appears to be a "debugging" statement, which should be avoided.  Too often statements like this are added during development, then forgotten -- the next developer won't know for sure if this fulfills a requirement, or is just a "tool" for development.